test-pom/pom.xml

```
@@ -64,7 +64,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>jenkins-test-harness</artifactId>
-      <version>2.54</version>
+      <version>2.55-rc1222.dbdd2bc9bc9b</version>
```

```suggestion
      <version>2.55</version>
```

Released it few minutes ago